Firmware/xyzcal.cpp

```diff
@@ -135,9 +135,11 @@ pos_mm_t pos_2_mm(float pos){
 	return pos * 0.01f;
 }
 
-void xyzcal_meassure_enter(void)
+void xyzcal_meassure_center(void)
 {
-	DBG(_n("xyzcal_meassure_enter\n"));
+	DBG(_n("xyzcal_meassure_center\n"));
+	lcd_set_cursor(4,3);
+	lcd_printf_P(PSTR("Measure center  ")); ////MSG_MEASURE_CENTER c=16
```

Comment:
Why is this one not translated?

Reply:
Try to avoid translations on rare used messages to save space.

Reply:
Please use lcd_puts_at_P if no formatting is needed.